runway/config.py

```diff
@@ -135,7 +135,7 @@ def __iter__(self):
 class ModuleDefinition(ConfigComponent):  # pylint: disable=too-many-instance-attributes
     """A module defines the directory to be processed and applicable options.
 
-    It can consist of `CloudFormation`_ (using `CFNgin`_),
+    It can consist of `CloudFormation`_ (using `CFNgin <terminology.html#runway-s-cfngin>`_),
```

Comment:
this should be fixed by correcting the `CFNgin` target in `docs/source/runway_config.rst` instead so it corrects all links on the page.

Reply:
Updated